core/src/components/card/test/shape/card.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Shape is only available in the Ionic theme.
+ */
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('card: shape'), () => {
+    test.describe('default', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.goto(`/src/components/card/test/shape`, config);
```

Comment:
Optional: Since we are going to the same page for every test, we can set this goto we can set this in a [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each) inside of the `chip: shape` describe block.